hnn_core/feed.py

```diff
@@ -7,8 +7,47 @@
 import numpy as np
 
 
-class ExtFeed(object):
-    """The ExtFeed class of external spike input times.
+# based on cdf for exp wait time distribution from unif [0, 1)
+# returns in ms based on lamtha in Hz
+def _t_wait(prng, lamtha):
+    return -1000. * np.log(1. - prng.rand()) / lamtha
+
+
+def _get_prng(seed, gid, sync_evinput=False):
+    """Random generator for this instance.
+
+    Parameters
+    ----------
+    seed : int
+        The seed for random state generator.
+    gid : int
+        The cell ID
+    sync_evinput : bool
+        If True, all cells get the same prng
+
+    Returns
+    -------
+    prng : instance of RandomState
+        The seed for events assuming a given start time.
+    prng2 : instance of RandomState
+        The seed for start times.
```

Comment:
I'm a little confused on the use of these two prng values. Could we clarify them a little in the documentation?

Reply:
to be honest, I am a bit confused myself. I thought `prng2` is used to get the start time jitter but actually it turns out if `t0 = -1` then `prng` is used. See: https://github.com/jonescompneurolab/hnn-core/blob/56f3565dcee8592872ac79a911b250c830d1d67b/hnn_core/feed.py#L225-L230

Nonetheless I clarified in the documentation assuming the intent was actually for the start time for both cases.

Reply:
It looks like `prng2` is only used for rhythmic inputs and defines the jittered start time of the rhythmic burst. Then, `prng` is used to define the jittered input of single input within that burst. If my understanding is correct, we may want to simplify `_get_prng()` by elimnating any reference to `prng2` and then adding the logic for creating a second random state variable to `feed_event_times()` only if a rhythmic (i.e., common) input is in the making.

Reply:
Actually, this will probably get resolved when we work on a more flexible API for creating new feeds so let's ignore this for now.

Reply:
That makes sense. Note though the current "API". Every function has the last argument as `prng`. This is how it should be. The user should be able to supply their own `prng`. It shouldn't be generated opaquely inside a function or class